### ISSUE TITLE
Variable substitutions

### ISSUE THREAD

**Issue by wtosta** [cifsd-team/ksmbd-tools#215]
Are you going to support variable substitutions just like they appear in Samba's smb.conf ? For example the option “path = /tmp/%u” is interpreted as “path = /tmp/john” I mean only the basic ones. E.g. %G, %U, %m, As you know these variables are very useful when you need to set up users home folders, which can be mapped as letters afterwards.

**Comment by namjaejeon** [comment]
Yep, It would be better if you could develop such a function and participate in ksmbd development.

**Comment by wtosta** [comment]
I asked politely if such functionality could be added. If not, you can normally say no or it is too complicated in a current situation. Leave your patronizing irony to yourself, "master of programming."

**Comment by namjaejeon** [comment]
If I had time, I could do everything. Look at the issues. I'm having a hard time getting it all done. This is open source. And I'm politely suggesting that you can implement the functionality you need yourself.

**Comment by wtosta** [comment]
I am not expressing any demands. You write the code for free and for that reason you have my respect. If you don't have any plans, e.g. due to lack of time, to write one or another functionality, that's fine. I understand it perfectly. Maybe there will be time someday, maybe not. It is only your will. Best regards :-)

**Comment by namjaejeon** [comment]
Thanks for your understanding! I will put it into my TODO list:). since ksmbd is merged into linux mainline, many issues have been coming. I understand the importance of the features you suggested. It will take some time, but please be patient.
